requirements/edx/github.txt

```diff
@@ -29,10 +29,10 @@ git+https://github.com/mitocw/django-cas.git@60a5b8e5a62e63e0d5d224a87f0b489201a
 -e git+https://github.com/edx/bok-choy.git@82d2f4b72e807b10d112179c0a4abd810a001b82#egg=bok_choy
 -e git+https://github.com/edx-solutions/django-splash.git@7579d052afcf474ece1239153cffe1c89935bc4f#egg=django-splash
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
--e git+https://github.com/edx/edx-ora2.git@release-2015-04-07T13.05#egg=edx-ora2
+-e git+https://github.com/singingwolfboy/edx-ora2.git@update-pytz#egg=edx-ora2
```

Comment:
@singingwolfboy I'm not sure I understand.  Is there an edx-ora2 PR that I don't know about? Why not merge the changes into edx-ora2 repo first, then bubble up the changes to edx-platform with a new release of the edx/edx-ora2?